tests/test_server_ops.py

```diff
@@ -403,6 +404,12 @@ async def test_server_ops_no_cleartext(self):
                 self.assertIn('location', resp_headers)
                 self.assertTrue(
                     resp_headers['location'].startswith('https://'))
+
+                self.assertIn('strict-transport-security', resp_headers)
+                # By default we enforce HTTPS via HSTS on all routes.
+                self.assertEqual(
+                    resp_headers['strict-transport-security'],
+                    'max-age=31536000; includesubdomains; preload')
```

Comment:
I don't think we should force `includeSubdomains` and `preload` here. We have no business in enforcing HSTS opinions on the entire domain on which EdgeDB is running.

Reply:
good catch, makes sense

Reply:
Fixed